src/services/data_model_engine.js

```diff
@@ -103,10 +104,25 @@ export default class DataModelEngine {
     return asset;
   }
 
+  async selectAssetsByIdentifier(identifiers, accessLevel) {
```

Comment:
`identifiers` => `identifier` as this may be misunderstood as taking multiple identifiers

Reply:
Also the name should be changed to reflect we are returning AssetIDS and not assets directly.